### PR TITLE
dmidecode: add cpu and memory information

### PR DESCRIPTION
**Pull request by agnivade**
- Removed the system information as it seems to be less
important than cpu and memory

----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/dmidecode.md [hunk 1]
```diff
@@ -15,10 +15,14 @@
 
 `sudo dmidecode -s system-serial-number`
 
-- Show all the BIOS information:
+- Show BIOS information:
```
Comment (sbrl) [inline pages/linux/dmidecode.md]: Perhaps we could be a bit more description as to the kind of things that the BIOS information might contain? I'm not sure.

Comment (agnivade) [reply pages/linux/dmidecode.md]: Things like which BIOS features are available - PCI, PNP, vendor name, vendor version. Do you want to add these ?

Comment (sbrl) [reply pages/linux/dmidecode.md]: Perhaps `(includes BIOS features and system information)` or something?

Comment (agnivade) [reply pages/linux/dmidecode.md]: Hmm .. actually system information has another tag `system`. :smile:  Do you want me to include both tags in the example ?

Comment (sbrl) [reply pages/linux/dmidecode.md]: Ah, let's leave it then :P

File: pages/linux/dmidecode.md [hunk 2]
```diff
@@ -15,10 +15,14 @@
...
 
 `sudo dmidecode -t bios`
 
-- Show all the system information:
+- Show CPU information:
 
-`sudo dmidecode -t system`
+`sudo dmidecode -t cpu`
```
Comment (jsonbruce) [inline pages/linux/dmidecode.md]: I'm not sure whether we have the different version of `dmidecode`.
But I do not have the `-t cpu` argument in my machine. only `dmidecode -t processor` available.

And below are the available `-t` arguments in my `dmidecode`:

```
➜  ~ dmidecode --version              
3.1

➜  ~ dmidecode -t

Valid type keywords are:
  bios
  system
  baseboard
  chassis
  processor
  memory
  cache
  connector
  slot
```

Comment (sbrl) [reply pages/linux/dmidecode.md]: @jsonbruce What OS and version are you using there?

Comment (pxgamer) [reply pages/linux/dmidecode.md]: I get this too with Ubuntu 17.10 (dmidecode v3.1).

Comment (agnivade) [reply pages/linux/dmidecode.md]: Thanks ! updated.

Comment (sbrl) [reply pages/linux/dmidecode.md]: Confirmed! Thanks for updating, @agnivade

**Pull request merged**